setup.py

```diff
@@ -43,7 +43,7 @@
         'rmq': [
             'pika>=1.0.0b2',
             'topika>=0.2.0, <0.3.0',
-            'tornado>=4; python_version<"3"',
+            'tornado<6; python_version<"3"',
```

Comment:
wait, so tornado 5 is ok for python<3 but not for python>=3?

Reply:
Yes it is a bit weird, but the reason is that for python 3, tornado 5 switches the default to `asyncio` for the event loop instead of their own. But `kiwipy` requires the tornado solution.

Reply:
Btw, if possible in the future, it would be nicer to start a review and submit all comments in one go at the end, instead of one comment per time, because I get an email per comment. So I just got about 20 emails :)

Reply:
Ah sorry - for me those are anyhow always grouped in one thread in gmail.
The downside of doing it the "review way" is that nobody besides me can see the comments until the review is submitted (so one needs to make sure to have the time to cover everything).
Anyhow, I'll make sure not to bother you again ;-)

Reply:
tornado 5 is not ok for python3 because of this 
 https://github.com/aiidateam/kiwipy/issues/9 .
I make PR https://github.com/aiidateam/kiwipy/pull/24 for this issue, but I not sure it is the right way.